.github/workflows/devcontainer-feature-release.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   deploy:
     needs: 
      - call-workflow-test
-    if: ${{ github.ref == 'refs/heads/main' }}
+    # if: ${{ github.ref == 'refs/heads/main' }}
```

Comment:
Is this temp for the PR?

Reply:
Yeah, that was just to enable deploying the feature from my fork and test it.
I have reverted it